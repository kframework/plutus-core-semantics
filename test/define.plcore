(program

(module Foo (imported) (exported () ()) (local () ())
  (define four (builtin addInteger 2 2))

  (define result (builtin multiplyInteger Foo.four Foo.four))
)

)
