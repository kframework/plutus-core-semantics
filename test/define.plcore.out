<T> <k> . </k> <lenv> .Map </lenv> <genv> ( Foo . result ) |-> 16 ( Foo . four ) |-> 4 </genv> <mod> Foo </mod> <sym> .Map </sym> <gas> 9996 </gas> </T>
